src/componentes/mounted/busca-ativa/citopatologico/Equipe/TabelaEquipeSemExame.jsx

```
@@ -30,118 +30,85 @@ const IDFiltrosOrdenacaoCito = {
+    tabelaDataEquipe,
+    liberarPesquisa,
+    tabelaData,
+    setTabelaData,
+    mixpanel,
+    aba,
+    sub_aba,
+    showSnackBar,
+    setShowSnackBar,
+    setFiltros_aplicados,
+})=>{
+    const tabelaDataEquipeSemExame = tabelaDataEquipe?.filter(item=>item.id_status_usuario != 12)
```

Biome (1.9.4)

[error] Use !== instead of !=. != is only allowed when comparing against `null`

!= is only allowed when comparing against null

Using != may be unsafe if you are relying on type coercion
Unsafe fix: Use !==


(lint/suspicious/noDoubleEquals)